docs/multicluster/user-guide.md

```diff
@@ -396,18 +396,12 @@ $kubectl annotate node node-1 multicluster.antrea.io/gateway=true
 ```
 
 Multi-cluster Controller in the member cluster will detect the Gateway Node, and
-create a `Gateway` CR with the same name as the Node. You can check the `Gateway`
-with: `kubectl get gateway node-1 -o yaml`, which should show the Gateway
-information like:
+create a `Gateway` CR with the same name as the Node.
```

Comment:
Here, I feel better to keep "You can check it with command:"

Reply:
Done